parsers/BUILD

```diff
@@ -7,6 +7,7 @@ cc_library(
     deps = [
         "//common:engine_filesystem_utils",
         "//datamodel:engine_datamodel_geometry",
+        "//datamodel:engine_datamodel_math",
```

Comment:
Why?

Reply:
Needed due to build fails for debug build, issue is still in research.

Reply:
Well it might be usefull to explicitly depend on it.